tests/CodeFormatterTests.cs

```diff
@@ -114,5 +114,26 @@ public async Task FSharpProjectsDoNotCreateException()
             Assert.Equal(0, formatResult.FilesFormatted);
             Assert.Equal(0, formatResult.FileCount);
         }
+
+       [Fact]
+        public async Task OnlyFormatFilesFromList()
+        {
+            var logger = new TestLogger();
+            var path = Path.GetFullPath("tests/projects/for_code_formatter/unformatted_project_with_more_files/unformatted_project.csproj", SolutionPath);
```

Comment:
Instead of adding another project, let's add a new file to the unformatted project and update the tests.

Reply:
Done!